packages/cadl-ranch-specs/http/payload/multipart/main.tsp

```diff
@@ -303,4 +308,35 @@ Content-Type: image/jpg
     @header contentType: "multipart/form-data",
     @body body: MultiPartRequest,
   ): NoContentResponse;
+
+  @scenario
+  @scenarioDoc("""
+Expect request (
+- according to https://datatracker.ietf.org/doc/html/rfc7578#section-4.4, content-type of file part shall be labeled with 
+  appropriate media type, cadl-ranch will check it; content-type of other parts is optional, cadl-ranch will ignore it.
+- according to https://datatracker.ietf.org/doc/html/rfc7578#section-4.2, filename of file part SHOULD be supplied.
+  If there are duplicated filename in same filedName, cadl-ranch can't parse them all.
+):
+```
+POST /upload HTTP/1.1
+Content-Length: 428
+Content-Type: multipart/form-data; boundary=abcde12345
+
+--abcde12345
+Content-Disposition: form-data; name="id"
+Content-Type: text/plain
+
+123
+--abcde12345
+Content-Disposition: form-data; name="profileImage"; filename="<any-name-is-ok>"
+Content-Type: application/octet-stream;
+
+{…file content…}
+--abcde12345--
+```
+""")
+  @doc("Test content-type: multipart/form-data")
+  @post
+  @route("/anonymous-model")
+  op anonymousModel(@header contentType: "multipart/form-data", ...MultiPartAnonymousRequest): NoContentResponse;
```

Comment:
I still don't see why `id` needs to get in there.

Can we do this?
```suggestion
  op anonymousModel(@header contentType: "multipart/form-data", profileImage: MultiPartRequest.profileImage): NoContentResponse;
```

Fine to wait for opinion from others. Not a strong opinion here.

Reply:
+1 with @weidongxu-microsoft

Reply:
Updated